bridge/client/app/_components/ktb-sequence-list/ktb-sequence-list.component.html

```diff
@@ -19,21 +19,24 @@ <h2>Sequences</h2>
         </div>
       </ng-container>
       <ng-template #trace>
-        <ng-container *ngIf="isTrace(row) as trace">
+        <ng-container *ngIf="isSubsequence(row) as subSequence">
           <dt-icon
-            *ngIf="!trace.isLoading() || trace.isApproval() || trace.getLastTrace().isApproval(); else loading"
+            *ngIf="subSequence.state === 'finished' || subSequence.hasPendingApproval; else loading"
```

Comment:
I would introduce a method `isFinished()` here. It is easier to read and also reduces necessary changes if the model or the result changes. And we have a `SequenceState` enum that can be used.